### PR TITLE
Added Longest Palindromic Substring

### LINKED ISSUE #830: Longest Palindromic Substring in C++, Java, Python
Please assign this issue to me.

### PR DESCRIPTION
**Pull request by MastersAbh**
Fixes #830  

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Added Implementation of Finding longest palindromic substring of a given string in C++,Java and Python.

#### Languages Used:
- C++
- Java
- Python

#### Files Added:
- Longest_Palindromic_Substring.cpp
- Longest_Palindromic_Substring.java
- Longest_Palindromic_Substring.py

### CODE REVIEW

File: Longest_Palindromic_Substring/Longest_Palindromic_Substring.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    LONGEST PALINDROMIC SUBSTRING
+
+    The task is to find the longest substring of a given string that is palindromic.
+    A palindromic string is a string which is same when read from front or back.
+
+    eg. radar is same when read frontwards or backwards.
+
+*/
+
+import java.io.*;
```
Comment (tsah20) [inline Longest_Palindromic_Substring/Longest_Palindromic_Substring.java]: Please avoid using * in import statements

Comment (MastersAbh) [reply Longest_Palindromic_Substring/Longest_Palindromic_Substring.java]: Sure I'll remove that. Anything else ?

File: Longest_Palindromic_Substring/Longest_Palindromic_Substring.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    public static void main(String[] args) {
+        String str = "algorithmhtirrithm";
+        String ans = longest_palindromic_substr(str);
+        System.out.println("The longest palindromic substring is: "+ans);
+    }
+}
+/*
+Input : starting_player = 0,
+        piles = {3,4,1,7,6}
+Output: Player 0 will win.
```
Comment (tsah20) [inline Longest_Palindromic_Substring/Longest_Palindromic_Substring.java]: Please remove these lines. This  does not look as the test case for this problem

File: Longest_Palindromic_Substring/Longest_Palindromic_Substring.java (hunk 1 above)
Comment (tsah20) [reply Longest_Palindromic_Substring/Longest_Palindromic_Substring.java]: Still reviewing, I will add all the comments by tomorrow

**Pull request merged**